public/js/scripts.js

```diff
@@ -1,56 +1,93 @@
-function bookSearch(){
-
+function bookSearch () {
     //user input stored in var
     const find = document.getElementById('textName').value;
     document.getElementById('results').innerHTML = "";
+    asyncReq(find);
```

Comment:
Another way to do this without coupling the `asyncReq` and `successResult` could be return the promise from `asyncReq` and pass the result to `successResult `.

e.g. 
```JS
asyncReq(find).then(successResult)
```